style.css

```diff
@@ -28,34 +28,100 @@
     cursor: pointer;
 }
 
+.fullHistory span:hover {
+    color: gray;
+    cursor: pointer;
+}
+
+.trashButton i:hover {
+    color: red;
+    opacity: .9;
+    cursor: pointer;
+}
+
 .calcField {
-    width: 25%;
+    width: 84%;
     margin: 0 auto;
-    border-style: 1px solid;
 }
 
-.topCalc {
-    margin: 20px 0;
+.opField {
+    width: 15%;
+    margin: 0 auto;
+}
+
+.topCalc{
+    justify-content: center;
+    display: flex;
+}
+
+.output {
+    width: 40%;
+    display: flex;
+}
+
+.history {
+    width: 40%;
+    margin: 0 auto;
+    text-align: right;
+    font-size: 1em;
+    min-height: 25px;
+}
+
+.fullHistory {
+    width: 40%;
+    margin: 0 auto;
+    text-align: right;
+    font-size: 1.5em;
+    min-height: 60px;
+    border: solid 1px;
+    padding: 7px;
+}
+
+.fullHistory p {
+    margin: 5px;
+}
+
+.opEquals {
+    font-weight: bolder;
+    margin: 5px;
+}
+
+#calc{
+    text-align: right;
+    font-size: 2em;
+}
+
+#op{
```

Comment:
Make sure you go through and fix up the code style issues in this file, you can run `npm run stylelint` to see any issues, then you can fix them.

While you are at it please do `npm run lint-fix` as well to auto fix any JS linting issues, sorry no auto fix for style linting.

Reply:
I think I did this step correctly. I had no errors on the css file and few from the js file. Let me know if you see anything it missed.

Reply:
Sounds good, usually I would say put spaces between your css selectors and the curly braces... but as long as the linter is happy I think we can move on.